src/Tasks/Microsoft.NET.Build.Tasks/FindItemsFromPackages.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.NET.Build.Tasks
     /// set of Packages.
     /// </summary>
     /// <remarks>
-    /// Both Items and Packages are expected to have 'PackageName' and 'PackageVersion' 
+    /// Both Items and Packages are expected to have ('PackageName' and 'PackageVersion)' or ('NuGetPackageId' and 'NuGetPackageVersion')
```

Comment:
What is the difference between PackageName and NuGetPackageId?

Reply:
Different conventions were used in different places. Semantically, they are the same.